<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XLSXParser</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="_2" stretch="0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Список файлов, в которых будет проведено сравнение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="fileList">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Нежелательные символы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineTrash">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Например: .,_</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Эти фразы не будут учитываться:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTableWidget" name="phrasesTable">
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragOnly</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Добавить фразу (+)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Убрать фразу (-)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_loadFile">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Загрузить файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_deleteFile">
        <property name="text">
         <string>Удалить из списка</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_sort">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Анализ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Категории</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Настроить категории</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
